### ISSUE TITLE
Can it be used?

### ISSUE THREAD

**Issue by Deng-Xian-Sheng** [goplus/tools#2]
“First available release version.”？？
I feel puzzled!
I compiled it and got:
```
➜  goplsp ./main -h
The Go Language source tools.

Usage: gopls [flags] <command> [command-flags] [command-args]

gopls is a Go language server. It is typically used with an editor to provide
language features. When no command is specified, gopls will default to the 'serve'
command. The language features can also be accessed via the gopls command-line interface.

Available commands are:

main:
  serve : run a server for Go code using the Language Server Protocol
  version : print the gopls version information
  bug : report a bug in gopls
  api-json : print json describing gopls API
  licenses : print licenses of included software
```

**Comment by Deng-Xian-Sheng** [comment]
Does it detect file extensions?
It seems that the goplus plug-in does not have lsp support. I use the go plug-in and change the executable file of the lsp service to one that I compiled (the lsp of goplus)
Then I found out that it works like the LSP of go, completely without the appearance of goplus syntax!
It does not recognize the goplus syntax!

This is: "First available release version."??

I feel confused and even a little angry!

So, how can goplus be popularized?!
